Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/validation/core/DccValidationCache.kt

```diff
@@ -9,11 +8,11 @@ import javax.inject.Inject
 import javax.inject.Singleton
 
 @Singleton
-class DccCountryLocalCache @Inject constructor(
+class DccValidationCache @Inject constructor(
     @CertificateValidation private val cacheDir: File,
 ) {
     private val mutex = Mutex()
-    private val cacheFile = File(cacheDir, "dcc_country_cache_raw")
+    private val cacheFile = File(cacheDir, "dcc_validation_cache_raw")
```

Comment:
Maybe `dcc_validation_cache_countries_raw` ?
We'll probably not write the rules and country into the same file.

Reply:
changed it